lib/gui/models/supported-formats.js

```diff
@@ -131,6 +131,24 @@ SupportedFormats.service('SupportedFormatsModel', function() {
     return this.isSupportedImage(path.basename(image, `.${extension}`));
   };
 
+  /**
+   * @summary Check if an image seems a Windows image
+   * @function
+   * @public
+   *
+   * @param {String} image - image path
+   * @returns {Boolean} whether the image seems to be a Windows image
+   *
+   * @example
+   * if (SupportedFormatsModel.seemsWindowsImage('path/to/en_windows_7_ultimate_with_sp1_x86_dvd_u_677460.iso')) {
+   *   console.log('Looks like a Windows image');
+   * }
+   */
+  this.seemsWindowsImage = (image) => {
```

Comment:
BTW, can we change the argument name to `imagePath`? In `ImageSelectionController` we call the whole image object as `image`, so clarifying it would be really nice.